Jenkinsfile

```diff
@@ -478,11 +499,13 @@ lock(resource: "build-${params.STREAM}") {
             """)
 
             if (s3_stream_dir) {
+              def force = params.FORCE ? "--force" : ""
```

Comment:
And here, I don't think we want any `--force`.

It's implying there can be existing artifacts, but the only way that's possible is if there was a previous `cosa buildupload` for the same build ID. But if that happened, then it means that that previous run hadn't updated `builds.json` before uploading non-OSTree artifacts. But that's not possible because we update `builds.json` right after uploading the OSTree. Do you agree?

So ideally here I think I would have preferred a switch which instead asserts that the artifacts didn't yet exist. Because otherwise it signals something went really wrong. And again, like in the other comment, I think gating on the `FORCE` parameter isn't appropriate.

To clarify, we don't have to block this PR on adding that switch. But at least removing the use of `--force`.

Reply:
removed `--force`